news/2018-08-17-the-qat-gazette-july-2018.md

```diff
@@ -0,0 +1,97 @@
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+We're quite used to “busy months” here, as any regular readers will be aware, but as far as major, meta-redefining changes go, July might have been our biggest and busiest in quite some time!
+
+Inside we'll be bringing you updates on these major overhauls, from first steps in a new, more structured system for gauging Beatmap Nominator performance, through to massive Ranking Criteria changes governing what types of beatmap can and can't be pushed to the Ranked category. Keep reading if any of that sounds interesting to you (which naturally, we hope it does!)
+
+## Within the Beatmap Nominators
+
+Only one major development in the Beatmap Nominator Team for you all this month, though it's a significant one that will affect how we keep track of each Nominator's activity, behaviour and proficiency in the future. Naturally there's also news on new and departing members, so read on to find out more!
+
+### Beatmap Nominator Re-Evaluations
+
+Throughout July, we conducted a thorough review of activity, proficiency and behaviour for all current members of the Beatmap Nominators, in an attempt to ensure the quality of their collective work, and to provide help and guidance on how to improve where necessary. Depending on each Nominator's performance in these three areas, varying degrees of action were taken, ranging from promotion / retention of full Beatmap Nominator status for sufficient performance and above, to probationary Beatmap Nominator status for insufficient performance.
+
+Probationary status currently operates under a “one warning” system — this means that Nominators who were currently on, or had been previously been placed on probation for any given reason were dismissed from the team if found to be performing below expectation for a second time. Regardless of the reason for dismissal, we thank each Nominator for their service to the mapping and modding community and invite them to return in a later application cycle should our concerns be satisfactorily addressed.
+
+Feedback was provided by Quality Assurance Team members in each Nominator's respective mode(s), explaining the reasons for the decision taken, as well as indicating any areas for potential improvement if applicable.
+
+As a result of the Beatmap Nominator Re-Evaluations, 49/95 Nominators were retained as, or promoted to, full member status, 34/95 were retained or placed on probation, and 12/95 were removed from the group. Future evaluations will take place in conjunction with new Beatmap Nominator application phases on a six-weekly basis.
+
+### New Beatmap Nominator Promotions
+
+Whilst our existing Beatmap Nominators were being reviewed, we were pleased to welcome a wave of some new and returning circle inspectors to the team. Please offer warm welcome and congratulations to the following on their admission into the Beatmap Nominators:
```

Comment:
Whilst -> While (it's just a bit archaic)
offer warm welcome -> offer a warm welcome

Reply:
OED says while is somehow older so massive shrug, I like whilst personally

Changed the second one, and also in the previous phrase replaced welcome with "present" to avoid repetition.